WebRoot/js/zimbraAdmin/search/model/ZaSearch.js

```diff
@@ -670,6 +671,8 @@ ZaSearch.getObjectCounts = function(types, callback) {
         cmdParams.soapDoc = soapDoc;
         cmdParams.asyncMode = true;
         cmdParams.callback = dataCallback;
+        // BatchRequest with no element is sent when types array has ZaItem.ACCOUNT only.
```

Comment:
are we going to send an empty BatchRequest? we should try to avoid that

Reply:
I am not aware of UI changes more here. Is there any another way to do this?

Reply:
Can you reach out to any UI developer for checking this?